<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>formArama</class>
 <widget class="QDialog" name="formArama" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Detaylı Arama</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="simgeler.qrc" >:/new/prefix1/icons/filefind.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame_3" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbTarih" >
        <property name="text" >
         <string>Sipariş ve Teslim Tarihlerini Dahil Et</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QDateEdit" name="dateSiparis" >
          <property name="displayFormat" >
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDateEdit" name="dateTeslim" >
          <property name="displayFormat" >
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Sipariş Tarihi</string>
          </property>
          <property name="buddy" >
           <cstring>dateSiparis</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Teslim Tarihi</string>
          </property>
          <property name="buddy" >
           <cstring>dateTeslim</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbTarih2" >
        <property name="text" >
         <string>Tarihler Arasında Ara</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QDateEdit" name="dateBasla" >
          <property name="displayFormat" >
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelIle" >
          <property name="text" >
           <string>ile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateBitis" >
          <property name="displayFormat" >
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ara</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>İptal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame_4" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbSiparis" >
        <property name="text" >
         <string>Siparişi Detaylı Ara</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>&amp;Fatura No</string>
          </property>
          <property name="buddy" >
           <cstring>lineFatura</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="lineEk" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>&amp;Ek İstekler</string>
          </property>
          <property name="buddy" >
           <cstring>lineEk</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>S&amp;ipariş Türü</string>
          </property>
          <property name="buddy" >
           <cstring>lineSiparis</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lineSiparis" />
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="lineFatura" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbMusteri" >
        <property name="text" >
         <string>Müşteriyi Detaylı Ara</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QComboBox" name="comboSektor" />
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="lineAdres" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Sektör</string>
          </property>
          <property name="buddy" >
           <cstring>comboSektor</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Müşteri Adı</string>
          </property>
          <property name="buddy" >
           <cstring>lineMusteri</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="lineMusteri" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>&amp;Adres</string>
          </property>
          <property name="buddy" >
           <cstring>lineAdres</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>cbMusteri</tabstop>
  <tabstop>comboSektor</tabstop>
  <tabstop>lineMusteri</tabstop>
  <tabstop>lineAdres</tabstop>
  <tabstop>cbSiparis</tabstop>
  <tabstop>lineSiparis</tabstop>
  <tabstop>lineFatura</tabstop>
  <tabstop>lineEk</tabstop>
  <tabstop>cbTarih</tabstop>
  <tabstop>dateSiparis</tabstop>
  <tabstop>dateTeslim</tabstop>
  <tabstop>cbTarih2</tabstop>
  <tabstop>dateBasla</tabstop>
  <tabstop>dateBitis</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="simgeler.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>formArama</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>478</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMusteri</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboSektor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMusteri</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineMusteri</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMusteri</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineAdres</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>165</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSiparis</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSiparis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSiparis</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineFatura</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSiparis</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEk</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTarih</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateSiparis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>167</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTarih</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTeslim</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>167</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTarih2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateBasla</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>361</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>305</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTarih2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateBitis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>361</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>418</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
